src/components/Exchange/ExchangeTVChart.js

```diff
@@ -440,31 +437,35 @@ export default function ExchangeTVChart(props) {
               />
             </div>
           </div>
-					<div>
-						<div className="ExchangeChart-main-price">{chartToken.maxPrice && formatAmount(chartToken.maxPrice, USD_DECIMALS, 2)}</div>
-						<div className="ExchangeChart-info-label">${chartToken.minPrice && formatAmount(chartToken.minPrice, USD_DECIMALS, 2)}</div>
-					</div>
-					<div>
-						<div className="ExchangeChart-info-label">24h Change</div>
-						<div className={cx({ positive: deltaPercentage > 0, negative: deltaPercentage < 0 })}>
-							{!deltaPercentageStr && "-"}
-							{deltaPercentageStr && deltaPercentageStr}
-						</div>
-					</div>
-					<div className="ExchangeChart-additional-info">
-						<div className="ExchangeChart-info-label">24h High</div>
-						<div>
-							{!high && "-"}
-							{high && high.toFixed(2)}
-						</div>
-					</div>
-					<div className="ExchangeChart-additional-info">
-						<div className="ExchangeChart-info-label">24h Low</div>
-						<div>
-							{!low && "-"}
-							{low && low.toFixed(2)}
-						</div>
-					</div>
+          <div>
+            <div className="ExchangeChart-main-price font-mono">
```

Comment:
maybe add that font to `.ExchangeChart` class in styles file? at first glance adding `font-mono` to different elements in views looks like a unnecessary duplication